README.md

```diff
@@ -1,11 +1,17 @@
 # SecureKeychain
 
+KeychainAccess is a simple Swift wrapper for Keychain that works on iOS and OS X. Makes using Keychain APIs extremely easy and much more palatable to use in Swift.
```

Comment:
Change `Makes using Keychain APIs extremely easy and much more palatable to use in Swift.`
to
`It makes using Keychain APIs extremely easier and much more comfortable using in Swift`